README.md

```diff
@@ -190,7 +190,7 @@ Feel free to check the [issues page][issues-url].
 
 Icon from:
```

Comment:
Could you change please the word "Icon" for "Icons"?, I've checked and the project is using 2 icons, make more sense to use "icons" instead

![image](https://user-images.githubusercontent.com/66505715/169298368-4edc05ec-48bd-411d-9165-f6b107a4f89a.png)

Reply:
yes, no problem